wcfsetup/install/files/lib/system/file/processor/FileProcessor.class.php

```diff
@@ -240,6 +240,11 @@ public function delete(array $files): void
 
     public function hasReachedUploadLimit(IFileProcessor $fileProcessor, array $context): bool
     {
+        // Replace button from `FileProcessorFormField` is clicked.
+        if (!empty($context["__replace"])) {
+            return false;
+        }
```

Comment:
```suggestion
        $isReplacement = $context['__replace'] ?? false;
        if ($isReplacement) {
            return false;
        }
```

Do not use `empty()`, it does not make clear if the key may be absent or if you’re simply checking if the value is falsy. I much prefer being verbose here. Even better: If the key may be absent, you can rewrite this while also getting rid of the comment.